src/openrct2-ui/windows/Multiplayer.cpp

```diff
@@ -300,7 +300,7 @@ static void window_multiplayer_draw_tab_images(rct_window* w, rct_drawpixelinfo*
 static void window_multiplayer_set_page(rct_window* w, int32_t page);
 
 static bool _windowInformationSizeDirty;
-static LocationXY16 _windowInformationSize;
+static CoordsXY _windowInformationSize;
```

Comment:
Same as above - not actually Coords, so don't abuse the struct.

Reply:
I dropped this commit.